mods/ts/sequences/structures.yaml

```diff
@@ -862,6 +862,10 @@ proc: # TODO: unused narefn_a, narefn_b
 	make: ntrefnmk
 		Length: 20
 		ShadowStart: 20
+	dock: ntrefn_a # TODO: unused
```

Comment:
It didn't even ship one. We added the sprite ourselves. https://github.com/OpenRA/OpenRA/commit/87d2d607e633938da80a7fd460a3c972d3354b3c